packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -388,12 +506,12 @@ Pagination.propTypes = {
    * The choices for `pageSize`.
    */
   pageSizes: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.number),
+    PropTypes.arrayOf(PropTypes.number.isRequired),
     PropTypes.arrayOf(
       PropTypes.shape({
-        text: PropTypes.text,
-        value: PropTypes.number,
-      })
+        text: PropTypes.string.isRequired,
+        value: PropTypes.number.isRequired,
+      }).isRequired
```

Comment:
Since I updated text -> string, and needed to add some isRequired to fields to prevent typescript errors, I needed to update the public api snapshot accordingly. Not sure if this will cause this to needed to go out in a major change.

Reply:
Cool. I don't think it will a problem!